src/test/java/net/infordata/em/TerminalClientTest.java

```diff
@@ -290,4 +282,182 @@ public void shouldSendExceptionToExceptionHandlerWhenLoginAndServerDownByLabel()
     exceptionWaiter.awaitException();
   }
 
+  @Test
+  public void shouldGetCorrectFieldsWhenGetFields() throws Exception {
+    awaitKeyboardUnlock();
+   List<TestField> actualFields = client.getFields().stream().map(TestField::new).collect(Collectors.toList());
+    assertEquals(buildExpectedFields(), actualFields);
+  }
+
+  private List<TestField> buildExpectedFields() {
+    List<TestField> testFieldList = new ArrayList<>();
+    testFieldList.add((new TestField(new TestField.FieldBuilder(FIELD_STRING_OUTPUT).withAttr(36).withDefaultColumn().withRow(5))));
```

Comment:
Since the value is not obvious and particular of this screen, I think is preferable to store it in a variable `int fieldsColumn = 52` and then use it with `withColumn(fieldsColumns)`.